scripts/ubuntu-16.04_tkgplus-1.17_weave-2.5.2_rev3/cust.sh

```diff
@@ -38,42 +48,42 @@ while [ `systemctl is-active docker` != 'active' ]; do echo 'waiting for docker'
-tar xvzf vmware-kubernetes-v1.17.3+vmware.1.tar.gz
-dpkg -i vmware-kubernetes-v1.17.3+vmware.1/debs/*.deb || :
+tar xvzf $vmware_kubernetes_dir_name.tar.gz
+dpkg -i $vmware_kubernetes_dir_name/debs/*.deb || :
 sudo apt-get -f install -y
 
 # kube proxy
-docker load -i ./vmware-kubernetes-v1.17.3+vmware.1/kubernetes-v1.17.3+vmware.1/images/kube-proxy-v1.17.3_vmware.1.tar.gz
+docker load -i ./$vmware_kubernetes_dir_name/$kubernetes_sub_dir/kube-proxy-v1.17.3_vmware.1.tar.gz
 
 # kube controller manager
-docker load -i ./vmware-kubernetes-v1.17.3+vmware.1/kubernetes-v1.17.3+vmware.1/images/kube-controller-manager-v1.17.3_vmware.1.tar.gz
+docker load -i ./$vmware_kubernetes_dir_name/$kubernetes_sub_dir/kube-controller-manager-v1.17.3_vmware.1.tar.gz
 
 # kube api server
-docker load -i ./vmware-kubernetes-v1.17.3+vmware.1/kubernetes-v1.17.3+vmware.1/images/kube-apiserver-v1.17.3_vmware.1.tar.gz
+docker load -i ./$vmware_kubernetes_dir_name/$kubernetes_sub_dir/kube-apiserver-v1.17.3_vmware.1.tar.gz
 
 # kube scheduler
-docker load -i ./vmware-kubernetes-v1.17.3+vmware.1/kubernetes-v1.17.3+vmware.1/images/kube-scheduler-v1.17.3_vmware.1.tar.gz
+docker load -i ./$vmware_kubernetes_dir_name/$kubernetes_sub_dir/kube-scheduler-v1.17.3_vmware.1.tar.gz
 
 # pause
-docker load -i ./vmware-kubernetes-v1.17.3+vmware.1/kubernetes-v1.17.3+vmware.1/images/pause-3.1.tar.gz
+docker load -i ./$vmware_kubernetes_dir_name/$kubernetes_sub_dir/pause-3.1.tar.gz
 
 # e2e test
-docker load -i ./vmware-kubernetes-v1.17.3+vmware.1/kubernetes-v1.17.3+vmware.1/images/e2e-test-v1.17.3_vmware.1.tar.gz
+docker load -i ./$vmware_kubernetes_dir_name/$kubernetes_sub_dir/e2e-test-v1.17.3_vmware.1.tar.gz
 
 # etcd
-docker load -i ./vmware-kubernetes-v1.17.3+vmware.1/etcd-v3.4.3+vmware.3/images/etcd-v3.4.3_vmware.3.tar.gz
+docker load -i ./$vmware_kubernetes_dir_name/$etcd_sub_dir/etcd-v3.4.3_vmware.3.tar.gz
 
 # coredns
-docker load -i ./vmware-kubernetes-v1.17.3+vmware.1/coredns-v1.6.5+vmware.3/images/coredns-v1.6.5_vmware.3.tar.gz
+docker load -i ./$vmware_kubernetes_dir_name/$coredns_sub_dir/coredns-v1.6.5_vmware.3.tar.gz
 
-docker tag vmware.io/kube-proxy:v1.17.3_vmware.1 k8s.gcr.io/kube-proxy:v1.17.3-vmware.1
-docker tag vmware.io/kube-controller-manager:v1.17.3_vmware.1 k8s.gcr.io/kube-controller-manager:v1.17.3-vmware.1
-docker tag vmware.io/kube-apiserver:v1.17.3_vmware.1 k8s.gcr.io/kube-apiserver:v1.17.3-vmware.1
-docker tag vmware.io/kube-scheduler:v1.17.3_vmware.1 k8s.gcr.io/kube-scheduler:v1.17.3-vmware.1
-docker tag vmware.io/pause:3.1 k8s.gcr.io/pause:3.1
-docker tag vmware.io/e2e-test:v1.17.3_vmware.1 k8s.gcr.io/e2e-test:v1.17.3-vmware.1
-docker tag vmware.io/etcd:v3.4.3_vmware.3 k8s.gcr.io/etcd:3.4.3-0
-docker tag vmware.io/coredns:v1.6.5_vmware.3  k8s.gcr.io/coredns:1.6.5
+docker tag vmware.io/kube-proxy:v1.17.3_vmware.1 k8s.gcr.io/kube-proxy:$kubernetes_version
```

Comment:
Can we keep the `vmware.io/` here so that the customer is clear about the source of the images. So we have:
```
docker tag vmware.io/kube-proxy:v1.17.3_vmware.1 vmware.io/kube-proxy:$kubernetes_version
```

Similarly in places with `registry.tkg.vmware.run`.